ocs_ci/deployment/ibmcloud.py

```diff
@@ -287,6 +301,99 @@ def _verify_volumes_deleted(resourece_group):
 
             _verify_volumes_deleted(resource_group)
 
+    def delete_leftover_resources(self, resource_group):
+        """
+        Delete leftovers from IBM Cloud.
+
+        Args:
+            resource_group: Resource group in IBM Cloud that contains the cluster resources.
+
+        """
+
+        def _get_resources(resource_group):
+            """
+            Return a list leftover resources for the specified Resource Group
+            """
+            cmd = f"ibmcloud resource service-instances --type all -g {resource_group} --output json"
+            proc = exec_cmd(cmd)
+
+            return json.loads(proc.stdout)
+
+        def _get_reclamations(resource_group):
+            """
+            Get reclamations for resource group.
+
+            Args:
+                rsource_group (str): Resource group name
+
+            Returns:
+                list: Reclamations for resource group if found.
```

Comment:
this function does not return rg_reclamations

Reply:
missed to return ;) thanks for noticing